addons/l10n_ug/data/template/account.account-ug.csv

```diff
@@ -0,0 +1,259 @@
+340329,Other Water resources,340329,asset_fixed,,False
+340331,Radio Spectrum - Other Natural Resources,340331,asset_fixed,,False
+340332,Aviation Airspace,340332,asset_fixed,,False
+340341,Mountains and Rocks,340341,asset_fixed,,False
+340349,Other Natural Resources (Not classified elsewhere),340349,asset_fixed,,False
+340401,"Marketable operating leases - Contracts, leases and Permits",340401,asset_non_current,,False
+340402,"Permits to use natural resources - Contracts, leases and Permits",340402,asset_non_current,,False
+340403,"Permits to undertake specific activities - Contracts, leases and Permits",340403,asset_non_current,,False
+340405,Entitlement to future goods and services on an exclusive basis - Contracts leases and Permits,340405,asset_non_current,,False
+340406,Goodwill and marketing assets,340406,asset_non_current,,False
+3510,Monetary Gold and SDRs,3510,asset_current,,False
+3522,Debt Securities,3522,asset_current,,False
+3523,Loans,3523,asset_current,,False
+352401,Shares in public corporations,352401,asset_non_current,,False
+352402,Shares in private entities,352402,asset_non_current,,False
+352501,Petroleum Revenue Investment Reserve,352501,asset_non_current,,False
+352599,Other Investment Fund Shares or Units,352599,asset_non_current,,False
+352701,Forwards,352701,asset_prepayments,,False
+352702,Futures,352702,asset_prepayments,,False
+352703,Options,352703,asset_prepayments,,False
+352704,Swaps,352704,asset_prepayments,,False
+3528,Account Receivable,3528,asset_receivable,,True
+352802,Staff Advances,352802,asset_receivable,,True
+352803,Other Advances,352803,asset_receivable,,True
+352804,Taxes Receivable,352804,asset_receivable,,True
+352805,Revenue receivable,352805,asset_receivable,,True
+352806,Trade debtors,352806,asset_receivable,,True
+352807,Sundry Debtors,352807,asset_receivable,,True
+352808,Pre-payments,352808,asset_receivable,,True
+352809,Deferred Expenses,352809,asset_receivable,,True
+4111,Currency Deposits,4111,liability_payable,,True
+411201,Treasury Bills,411201,liability_current,,False
+411202,Treasury Bonds,411202,liability_non_current,,False
+411203,Promissory Notes,411203,liability_current,,False
+411301,Commercial Banks and Other Financial Institutions,411301,liability_current,,False
+411302,From other Government Units,411302,liability_current,,False
+411303,From State Owned Enterprises,411303,liability_current,,False
+411304,Interest Payable,411304,liability_current,,False
+411315,Other Loans,411349,liability_current,,False
+4115,Investment Fund Shares or Units,4115,liability_non_current,,False
+411601,Forwards,411601,liability_current,,False
+411602,Futures,411602,liability_current,,False
+411603,Options,411603,liability_current,,False
+411604,Swaps,411604,liability_current,,False
+4117,Accounts Payable,4117,liability_payable,,True
+411711,Arrears,411711,liability_payable,,True
+411721,Trade creditors,411721,liability_payable,,True
+411722,Taxes payable,411722,liability_payable,,True
```

Comment:
This should be current liabilities and we should not have that many payable accounts.  (see below)  (and why do we have 2?)